src/org/traccar/database/DataManager.java

```diff
@@ -297,7 +297,7 @@ private void initDatabaseSchema() throws SQLException, LiquibaseException {
 
     public User login(String email, String password) throws SQLException {
         User user = QueryBuilder.create(dataSource, getQuery("database.loginUser"))
-                .setString("email", email)
+                .setString("email", email.trim().toLowerCase())
```

Comment:
1. database would allow not allow two users to be created with same address in different cases as it is using `utf8_general_ci` collation.
2. many users mess up with capslock on/off or capping certain letters in their email
3. this is standard practice with most sites to prevent user errors due to 2